app/src/lib/git/status.ts

```diff
@@ -256,6 +257,19 @@ function getConflictStatus(
     return { kind: 'text', conflictMarkerCount }
   }
 
+  if (status.kind === 'conflicted') {
+    const { us, them } = status
+    const conflictWithoutMarkers =
+      them !== GitStatusEntry.UpdatedButUnmerged &&
+      us !== GitStatusEntry.UpdatedButUnmerged &&
+      them !== GitStatusEntry.Modified &&
```

Comment:
I don't think this check is quite accurate, because it'll also catch situations where one branch has modified the file, but the other has not. 

My original repro passes fine if we make the check something like this:

```diff
diff --git a/app/src/lib/git/status.ts b/app/src/lib/git/status.ts
index 43615b893..2f5d842d5 100644
--- a/app/src/lib/git/status.ts
+++ b/app/src/lib/git/status.ts
@@ -259,11 +259,14 @@ function getConflictStatus(
 
   if (status.kind === 'conflicted') {
     const { us, them } = status
+
+    const bothFilesModified =
+      them === GitStatusEntry.Modified && us === GitStatusEntry.Modified
+
     const conflictWithoutMarkers =
       them !== GitStatusEntry.UpdatedButUnmerged &&
       us !== GitStatusEntry.UpdatedButUnmerged &&
-      them !== GitStatusEntry.Modified &&
-      us !== GitStatusEntry.Modified
+      !bothFilesModified
 
     if (conflictWithoutMarkers) {
       return { kind: 'text', conflictMarkerCount: null, us, them }
```

And with this check I see text files that we previously considered resolved show have conflicts that they shouldn't be handling in an editor:

<img width="1187" alt="screen shot 2018-11-09 at 10 29 56 am" src="https://user-images.githubusercontent.com/359239/48268188-b73f8780-e40a-11e8-8360-e72cb2b383f1.png">

Reply:
Actually, I like this change to the logic of how we detect them to also call out "both added" which also looks to put the content on-disk with merge conflicts (if any):

```
diff --git a/app/src/lib/git/status.ts b/app/src/lib/git/status.ts
index 43615b893..5222dd49f 100644
--- a/app/src/lib/git/status.ts
+++ b/app/src/lib/git/status.ts
@@ -259,11 +259,18 @@ function getConflictStatus(
 
   if (status.kind === 'conflicted') {
     const { us, them } = status
+
+    const bothFilesModified =
+      them === GitStatusEntry.Modified && us === GitStatusEntry.Modified
+
+    const bothFilesAdded =
+      them === GitStatusEntry.Added && us === GitStatusEntry.Added
+
     const conflictWithoutMarkers =
       them !== GitStatusEntry.UpdatedButUnmerged &&
       us !== GitStatusEntry.UpdatedButUnmerged &&
-      them !== GitStatusEntry.Modified &&
-      us !== GitStatusEntry.Modified
+      !bothFilesModified &&
+      !bothFilesAdded
 
     if (conflictWithoutMarkers) {
       return { kind: 'text', conflictMarkerCount: null, us, them }
```